dcompiler.dd

```diff
@@ -71,91 +71,192 @@ $(H2 $(LNAME2 files, Files))
     $(DD Sample D programs)
     )
     $(WINDOWS
+
+    $(DT $(D $(DMDDIR)\windows\bin\ddemangle.exe)
+    $(DD D symbol demangler)
+    )
+
+    $(DT $(D $(DMDDIR)\windows\bin\dman.exe)
+    $(DD D manual lookup tool)
+    )
+
     $(DT $(D $(DMDDIR)\windows\bin\dmd.exe)
     $(DD D compiler executable)
     )
 
+    $(DT $(D $(DMDDIR)\windows\bin\$(DUB))
+    $(DD D's package manager)
+    )
+
+    $(DT $(D $(DMDDIR)\windows\bin\$(DUMPOBJ))
+    $(DD Elf file dumper)
+    )
+
+    $(DT $(D $(DMDDIR)\windows\bin\$(DUSTMITE))
+    $(DD D source code minimizer)
+    )
+
+    $(DT $(D $(DMDDIR)\windows\bin\$(LINK2 http://www.digitalmars.com/ctg/optlink.html, link.exe))
+    $(DD OPTLINK)
+    )
+
+    $(DT $(D $(DMDDIR)\windows\bin\$(LINK2 http://www.digitalmars.com/ctg/make.html, make.exe))
+    $(DD Digitalmars Make)
+    )
+
+    $(DT $(D $(DMDDIR)\windows\bin\$(OBJ2ASM))
+    $(DD Elf file disassembler)
+    )
```

Comment:
Even though I now see this + is just a reordering, this is still wrong, isn't it? There is no `obj2asm.exe` in the Windows distribution, and seeing as how it's described as an "Elf file disassembler", it wouldn't make sense to include it. (Also, it's ELF, not Elf.)